vhost-user-backend/src/backend.rs

```diff
@@ -87,6 +89,15 @@ pub trait VhostUserBackend: Send + Sync {
     /// function.
     fn set_backend_req_fd(&self, _backend: Backend) {}
 
+    /// This function gets an owned file descriptor that the front-end can use otherwise
+    /// no file descriptor.
+    ///
+    /// A default implementation is provided as we cannot expect all backends to implement this
+    /// function.
+    fn get_shared_object(&self, _uuid: VhostUserSharedMsg) -> Result<Option<OwnedFd>> {
+        Ok(None)
```

Comment:
It is nice to have a default implementation, but I wonder why this differs from the `VhostUserBackendMut` counterpart implementation. IMO this could also fail with `Unsupported` error.

Reply:
yes, this was an oversight. thanks for pointing it out.